components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/exception/DashboardUnauthorizedException.java

```diff
@@ -0,0 +1,43 @@
+package org.wso2.carbon.dashboards.core.exception;
+
+/**
+ * Indicates an unauthorized exception occurred in dashboard.
+ */
+public class DashboardUnauthorizedException extends DashboardException {
```

Comment:
IMO we don't need ' Dashboard' prefix here, simply `UnauthorizedException` will be enough